src/Vulkan/Silk.NET.Vulkan/Enums/ImageLayout.gen.cs

```diff
@@ -55,7 +49,7 @@ public enum ImageLayout : int
         [NativeName("Name", "VK_IMAGE_LAYOUT_FRAGMENT_DENSITY_MAP_OPTIMAL_EXT")]
         FragmentDensityMapOptimalExt = 1000218000,
         [NativeName("Name", "VK_IMAGE_LAYOUT_FRAGMENT_SHADING_RATE_ATTACHMENT_OPTIMAL_KHR")]
-        FragmentShadingRateAttachmentOptimalKhr = 1000226003,
+        FragmentShadingRateAttachmentOptimalKhr = 1000164003,
```

Comment:
That's not good. Is this just khronos trying to find out how many people using fragment shaind rate or is this our mistake?

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkImageLayout.html

Reply:
I guess we used to convert this the wrong way. Resolved.